lib/src/main/java/com/diffplug/spotless/kotlin/KtLintStep.java

```diff
@@ -77,13 +83,22 @@ public static String defaultVersion() {
 
 		/** Are the files being linted Kotlin script files. */
 		private final boolean isScript;
+		private final String pkg;
 		/** The jar that contains the eclipse formatter. */
 		final JarState jarState;
 		private final TreeMap<String, String> userData;
 
 		State(String version, Provisioner provisioner, boolean isScript, Map<String, String> userData) throws IOException {
 			this.userData = new TreeMap<>(userData);
-			this.jarState = JarState.from(MAVEN_COORDINATE + version, provisioner);
+			String coordinate = MAVEN_COORDINATE;
```

Comment:
It's easier to read if you assign variables once rather than twice:

```
String coordinate;
if (blah) {
  coordinate = "x";
} else {
  coordinate = "y"
}
```